native/acter/src/api/pins.rs

```diff
@@ -224,8 +226,8 @@ impl Pin {
             .and_then(|t| t.formatted.clone().map(|f| f.body))
     }
 
-    pub fn content_text(&self) -> Option<String> {
-        self.content.content.clone().map(|t| t.body)
+    pub fn content_text(&self) -> Option<MsgContent> {
```

Comment:
this clearly has the wrong name now...
```suggestion
    pub fn content(&self) -> Option<MsgContent> {
```